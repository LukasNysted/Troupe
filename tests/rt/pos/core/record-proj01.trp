let fun f(r) = r.x
in f {x = 1} + f {y = 200, x = 2} + f {x = 300, x = 3} + f {x = 4, z = 400}
end