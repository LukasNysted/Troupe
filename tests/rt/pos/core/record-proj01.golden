2023-07-28T14:10:56.698Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: 10@{}%{}
